include/clang/Interpreter/CppInterOp.h

```diff
@@ -286,7 +286,16 @@ namespace Cpp {
 
   /// Gets a list of all the Methods that are in the Class that is
   /// supplied as a parameter.
-  CPPINTEROP_API std::vector<TCppFunction_t> GetClassMethods(TCppScope_t klass);
+  ///\param[in] klass - Pointer to the scope/class under which the methods have
+  /// to be retrieved
```

Comment:
The second line needs to be aligned with the start of `Pointer`. We should document the second parameter.

Reply:
Shouldn't it be aligned with the start of klass? I see that is what is followed in the rest of the code https://github.com/compiler-research/CppInterOp/blob/786ebeed27ad5c31d89e581ab1b5fd8cc9a230f0/include/clang/Interpreter/CppInterOp.h#L110-L114